addons/web/static/src/views/form/form_controller.js

```diff
@@ -153,9 +153,9 @@ export class FormController extends Component {
         // enable the archive feature in Actions menu only if the active field is in the view
         this.archiveEnabled =
             "active" in activeFields
-                ? !activeFields.active.readonly
+                ? !activeFields.active.modifiers.readonly
```

Comment:
In list, we read inside `fields` (`this.props.fields`) instead of `activeFields`. This is also what we did in legacy. So instead of reading inside `modifiers` here, I would replace `activeFields` by `this.props.fields` in this expression.